### PR TITLE
Add '--jerry-heap-section' option to build script

### PR DESCRIPTION
**Pull request by LaszloLango**
Based on previous work of Tamás Czene <tczene.u-szeged@partner.samsung.com> #307

IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by zherczeg** [comment]
LGTM

File: tools/build.py [hunk 1]
```diff
@@ -498,6 +500,10 @@ def build_libjerry(option):
         cmake_opt.append('-DEXTERNAL_MEM_HEAP_SIZE_KB=' +
                          str(option.jerry_heaplimit))
 
+    # --jerry-heap-section
+    if option.jerry_heap_section:
+        cmake_opt.append('-DJERRY_HEAP_SECTION_ATTR=\".sram_section\"')
```
Comment (akiss77) [inline tools/build.py]: Why is the value of the section hardcoded? Wouldn't it be better to have it coming as an argument?

Comment (LaszloLango) [inline tools/build.py]: Good idea. I'll update the patch.

**Comment by akiss77** [comment]
LGTM

**Pull request closed without merging**